code/modules/projectiles/guns/ballistic.dm

```diff
@@ -202,6 +202,7 @@
 		slot_flags &= ~ITEM_SLOT_BACK	//you can't sling it on your back
 		slot_flags |= ITEM_SLOT_BELT		//but you can wear it on your belt (poorly concealed under a trenchcoat, ideally)
 		sawn_off = TRUE
+		projectile_damage_multiplier = src.projectile_damage_multiplier - 0.1 //We lose 10% of damange if were spawn off
```

Comment:
this'll go negative if its got a damage mod of 0 to 0.09

Reply:
no gun is 0.09 damage mod

Reply:
something funny tho is if their is a gun that can be sawn that is smaller then normal, it will get bigger to be normal size

Reply:
No gun is 0.09 mod *right now.* correctly sanity check this, please.

Reply:
This is a non-issue, theirs no gun that can be sawn down thats 0.09 damage mod.

Reply:
Re-read my comment.

Reply:
you should be aiming to cover edge-cases like this, regardless of if they are already occurring or not, just make it the maximum value of the result and 0

Reply:
What's not sanity checked today will become a balance nightmare tomorrow. I for one, am hype to run up on a dying RD and loose healing lasers from a scatter laser shell into them.